front/src/components/common/ProtectedRoute.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store';
+import { Redirect } from 'react-router-dom';
+import { selectIsLoggedIn } from 'store/user/selectors';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+const ProtectedRoute: React.FC<Props> = ({ children }: Props) => {
+  const isUserLoggedIn = useSelector((state: AppState) => selectIsLoggedIn(state));
```

Comment:
I think you just pass in the selector, like in this file, but not a huge deal

https://github.com/trybick/tv-minder/blob/master/front/src/pages/MyShowsPage.tsx#L12